app-tasks/src/rf/layer_uploads/workflows.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+from botoflow.workflow_definition import WorkflowDefinition
+from botoflow.decorators import execute
+from botoflow.core import async
+
+from .activities import LayerImageUploadActivities
+from ..utils.logger import get_logger
+
+logger = get_logger()
+
+
+class LayerImageUploadWorkflow(WorkflowDefinition):
+    """Workflow that manages the upload of a layer image"""
+
+    @execute(os.environ.get('SWF_VERSION', '1.0'), execution_start_to_close_timeout=60)
+    def process_layer(self, image_path):
+        """Entrypoint to layer image upload workflow"""
+
+        logger.info('Starting Workflow to process %s', image_path)
```

Comment:
When I run a task I see this log message repeated 9 times rather than 3 as I would expect; is this a similar behavior to what we saw in the Ruby flow framework where decider functions would get re-run multiple times and we would use the `is_replaying?` flag to check if it was the first time through or not? Does botoflow have similar functionality?